scenarios/kubernetes_e2e.py

```diff
@@ -434,6 +434,12 @@ def main(args):
       'KUBE_GKE_NETWORK=%s' % cluster,
     )
 
+    if args.logexporter == 'true':
+        mode.add_environment(
+            'ENABLE_LOGEXPORTER=true',
+            'GCS_ARTIFACTS_DIR=%s' % args.gcs_artifacts_dir,
```

Comment:
We're in the process of migrating everything from magic environment variables which are hard to track down into more clear explicit config.

I would much prefer that any new state passed from kubernetes_e2e.py to kubetest uses a flag rather than an environment variable.

Examples:
* https://github.com/kubernetes/test-infra/issues/2829
* https://github.com/kubernetes/test-infra/issues/3375